store/sqlstore/channel_store.go

```diff
@@ -1492,11 +1492,9 @@ func (s SqlChannelStore) IsUserInChannelUseCache(userId string, channelId string
 	return false
 }
 
-func (s SqlChannelStore) GetMemberForPost(postId string, userId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var dbMember channelMemberWithSchemeRoles
-		if err := s.GetReplica().SelectOne(&dbMember,
-			`
+func (s SqlChannelStore) GetMemberForPost(postId string, userId string) (*model.ChannelMember, *model.AppError) {
+	var dbMember channelMemberWithSchemeRoles
+	query := `
 			SELECT
```

Comment:
nit: Probably 1 tab less in the query string would be more consistent with the rest of the indentation.

Reply:
done :)